libraries/tickv/src/async_ops.rs

```diff
@@ -555,9 +568,10 @@ mod store_flast_ctrl {
             _ => unreachable!(),
         }
 
-        let ret = tickv.append_key(get_hashed_key(b"TWO"), &VALUE);
+        #[allow(unsafe_code)]
```

Comment:
Could we have a `mod test` (within this file) to encapsulate all of these test functions, and then mark only that test module with `#[allow(unsafe_code)]`? That makes it much clearer that only the test code is allowed to use unsafe.